README.md

```diff
@@ -11,6 +11,10 @@ your application are configured.
 Once you have a Compose file, you can create and start your application with a
 single command: `docker compose up`.
 
+# About v2 update compatibility breaks
```

Comment:
that's a bit of a frightening title!
I suggest "About update and backward compatibility"

Reply:
lol, done